src/docs/components/DocsActiveBlock.jsx

```diff
@@ -61,17 +61,37 @@ class DocsActiveBlock extends Component {
     this.setState({renderNormally: false}, () => this.setState({renderNormally: true}));
   }
 
+  getPropsWithDefaults(component) {
+    const settings = this.props.settings;
+    let unchangedProps, propsWithDefaults;
+
+    if (Array.isArray(settings)) {
```

Comment:
what happen  if settings is not array?  returning empty array?
We can invert if for fast return values
```
if(!Array.isArray(settings)){
return; // undefined or should be here empty array 
}
rest of code
```